### DISCUSSION TITLE
Getting zero transverse momentum

### DISCUSSION THREAD

**Discussion by sbphysics** [MadAnalysis/madanalysis5#188] [Q&A]
Dear Madanalysis Team,
I am new to Madanalysis. I have generated Higgs boson and made it decay to some final state with Madgraph. I am trying to run the built in Madanalysis in Madgraph before passing the events through Pythia and Delphes and plot the transverse momentun of the intermediate Higgs boson. In this way I am not sure why I am getting zero transverse momentum for Higgs. How can I fix this? Any help will be appreciated.

Thanks in advance
SB

**Comment by jackaraz** [comment]
Hi @sbphysics, Higgs is not a final state particle hence after event reconstruction there is no such thing as "Higgs". You need to reconstruct its momentum using final state particles.
